orchestration/src/main/java/com/sap/ai/sdk/orchestration/DpiMasking.java

```diff
@@ -74,17 +77,40 @@ public DpiMasking withEntities(
       val entitiesList = new ArrayList<DPIEntities>();
       entitiesList.add(entity);
       entitiesList.addAll(Arrays.asList(entities));
-      return new DpiMasking(maskingMethod, entitiesList);
+      return new DpiMasking(maskingMethod, entitiesList, false, List.of());
     }
   }
 
+  /**
+   * If grounding is used, the input text will be masked.
+   *
+   * @return A new {@link DpiMasking} instance
+   */
+  @Nonnull
+  public DpiMasking withMaskGroundingEnabled() {
```

Comment:
(Question)

What was the decision **not** to use lombok wither?

Reply:
so you don't have to put `true`, plus defining Javadoc

Reply:
Then how do I disable the flag?

Reply:
I used `@With`